README.md

```diff
@@ -255,8 +255,14 @@ This tool is meant to be used as a quick-to-run, easily-automatable check in a
 non-malicious environment so that developers can be made aware of old libraries
 with security issues that have been compiled into their binaries.
 
+This tool only does a quick scan, because of which, the results may contain false negatives and/or false positives as well. This can be dealt with to a large extent by the use of a triage report.
+
+Moreover the tool does not guarantee that any vulnerabilities reported are actually present or exploitable.
+
```

Comment:
Let's use this paragraph to talk about false posititives/negatives and triage.  Here's a stab at some wording.

```suggestion
The tool does not guarantee that any vulnerabilities reported are actually present or exploitable.

Users can add triage information to reports to mark issues as false positives, false negatives, indicate that the risk has been mitigated by configuration/usage changes, and so on.  Triage details can be re-used on other projects so, for example, triage on a Linux base image could be applied to multiple containers using that image.
```

I think you need links to docs on how to add triage to a report and probably something on how to merge triage from an existing report here.   Not sure if we have a good guide on that particular workflow yet; if you're not sure the existing docs links are sufficient feel free to open a new issue for it.